### ISSUE TITLE
Version v1.8.0 not available in pkg.go.dev

### ISSUE THREAD

**Issue by luxifer** [lbrlabs/pulumi-scaleway#224]
Hi,

I'm using the plugin for some e2e tests @ Scaleway. Right now, the latest version v1.8.0 of the SDK does not seem to be usable.

See:
```
$ go get -u github.com/lbrlabs/pulumi-scaleway/sdk@v1.8.0
go: downloading github.com/lbrlabs/pulumi-scaleway v1.8.0
go: module github.com/lbrlabs/pulumi-scaleway@v1.8.0 found, but does not contain package github.com/lbrlabs/pulumi-scaleway/sdk
```

**Comment by luxifer** [comment]
It seems that the project is missing the `sdk/v1.8.0` tag.

**Comment by juchom** [comment]
Same issue with 1.8.1 :

```
pulumi plugin install resource scaleway v1.8.1 --server github://api.github.com/lbrlabs

error: [resource plugin scaleway-1.8.1] downloading from github://api.github.com/lbrlabs: failed to download plugin: scaleway-1.8.1: 404 HTTP error fetching plugin from https://api.github.com/repos/lbrlabs/pulumi-scaleway/releases/tags/v1.8.1. If this is a private GitHub repository, try providing a token via the GITHUB_TOKEN environment variable. See: https://github.com/settings/tokens
```

**Comment by dirien** [comment]
Fixed the issue with version `1.8.0`


```bash
➜ go get -u github.com/lbrlabs/pulumi-scaleway/sdk@v1.8.0
go: added github.com/lbrlabs/pulumi-scaleway/sdk v1.8.0
```

But I am working on a new version for the provider!
